src/Rings/groebner.jl

```diff
@@ -150,6 +153,22 @@ function standard_basis(I::MPolyIdeal; ordering::MonomialOrdering = default_orde
 		end
 	elseif algorithm == :fglm
 		_compute_groebner_basis_using_fglm(I, ordering)
+  elseif algorithm == :hilbert
+    if typeof(base_ring(I)) <: MPolyRing_dec
```

Comment:
Better:
```suggestion
    if base_ring(I) isa MPolyRing_dec
```
Potentially even better: use a helper function with two methods that does this, but only if it can be reused (so don't bother)